src/runtime/locale/en.ts

```diff
@@ -0,0 +1,38 @@
+export default {
+  name: 'English',
+  ui: {
+    inputMenu: {
+      noMatch: 'No matching data',
+      noData: 'No data'
+    },
+    commandPalette: {
+      noMatch: 'No matching data',
+      noData: 'No data',
+      close: 'Close'
+    },
+    selectMenu: {
+      noMatch: 'No matching data',
```

Comment:
Why change `No results` to `No matching data`?

Reply:
`No matching data` -  makes it very clear that we haven't managed to find and we don't need to put `for %s`.